src/imageops/mod.rs

```diff
@@ -164,6 +164,21 @@ where
     }
 }
 
+/// Tile an image by repeating it multiple times
+pub fn tile<I, J>(bottom: &mut I, top: &J)
+where
+    I: GenericImage,
+    J: GenericImageView<Pixel = I::Pixel>,
+{
+    for i in 0..(bottom.width() / top.width() + 1) {
+        let x = i * top.width();
```

Comment:
If both images have the same width then this loop runs twice, the second time not adding any pixels. You can use the iterator combinators to avoid this off-by-one:

```suggestion
    for x in (0..bottom.width()).step_by(top.width()) {
```